.github/workflows/ci.yaml

```diff
@@ -1,46 +0,0 @@
----
-
-name: ci
```

Comment:
Why do you delete this file?

Reply:
To reduce the amount of CI tasks, I've moved the `check` to run in `integrate`.